.coderabbit.yaml

```
@@ -0,0 +1,28 @@
+# CodeRabbit Configuration
+# このファイルはCodeRabbitの動作を設定します
+
+reviews:
+  # レビューの詳細度（assertive: 指摘多め, chill: 控えめ）
+  profile: chill
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)